packages/outline-docs/src/guides/development/component-development/06-properties.mdx

```diff
@@ -21,7 +21,7 @@ import '@phase2/outline-container';
   bottom-margin="spacing-8"
 >
   ## Description
-  Lit components receive input and store their state as JavaScript class fields or properties.
+  Outline components receive input and store their state as JavaScript class fields(normally called slots) or properties.
```

Comment:
did you mean normally called attributes?

Reply:
yeah i think that makes more sense